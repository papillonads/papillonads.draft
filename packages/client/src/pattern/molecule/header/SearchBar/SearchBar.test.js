import defaultRender from './SearchBar.render'

describe('<SearchBar />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
