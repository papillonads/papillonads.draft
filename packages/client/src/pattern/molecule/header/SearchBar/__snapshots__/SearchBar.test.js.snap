// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> Snaphot must match defaultRender 1`] = `null`;
